### PR TITLE
sci-mathematics/cvc4: fix build with bash 5.2

### PR DESCRIPTION
**Pull request by kcning**
See: https://salsa.debian.org/science-team/cvc4/-/merge_requests/2/diffs?commit_id=05ca9eee24e279ddfbaebea7393b4303200141ad
Closes: https://bugs.gentoo.org/883273

Signed-off-by: Kai-Chun Ning [kaichun.ning@gmail.com](mailto:kaichun.ning@gmail.com)

### CODE REVIEW

File: sci-mathematics/cvc4/files/cvc4-1.8-bash-5.2-fix.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+diff --git a/src/expr/mkexpr b/src/expr/mkexpr
```
Comment (thesamesam) [inline sci-mathematics/cvc4/files/cvc4-1.8-bash-5.2-fix.patch:1]: Please add links to:
1. any relevant Gentoo bugs
2. the patch source
at the top of the patch file.

Comment (kcning) [reply sci-mathematics/cvc4/files/cvc4-1.8-bash-5.2-fix.patch:9]: Done, I noticed that not all patches include a link to the Gentoo bug though. Can you kindly explain when it's needed and when it's not?

Comment (thesamesam) [reply sci-mathematics/cvc4/files/cvc4-1.8-bash-5.2-fix.patch:9]: It's kind of a newer thing that we do. See https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto.

It overall makes it easier to understand why a patch is there & if we still need it.

Comment (kcning) [reply sci-mathematics/cvc4/files/cvc4-1.8-bash-5.2-fix.patch:9]: ok, thank you

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**